File: go/intro.go
package main

import "fmt"

func main() {

	var a, s int // variable 

    fmt.Println("Example using variable in go \n")
	fmt.Println("Enter length of side : ")
	fmt.Scanf("%d", &s)


	fmt.Println("the formula a = s * s")
    a = s * s // calculate area; then store in a
    fmt.Println("\n")

    fmt.Println("Area of Square = ", a)
}

File: cpp/intro.cpp
#include <iostream>
#include <cstdio>
using namespace std;

// function type integer
int rectangle(int s) {
  return (s*s);
}

int main() {

    // variable 
    int a, s, threshold = 100;

    cout << "Example using variable in c++ \n" << endl;
    cout << "Enter length of side : ";

    scanf("%d", &s); // store length in s
    
    cout << "the formula a = s * s" << endl;
    a = rectangle(s); // calculate area; then store in a
    cout << endl;

    cout <<"\n";
    if (s > (threshold+1)) {
       cout <<"size:"<< s <<" more than threshold:"<< threshold;
    } else if ( s == 100) {
      cout <<"size:"<< s <<" more than threshold:"<< threshold;
    } else {
      cout <<"size:"<< s <<" more than threshold:"<< threshold;       
    }

    cout << endl;

    cout <<"Area of Square = " << a << endl;

    return 0;    
}